app/config/parameters.yml.dist

```diff
@@ -14,4 +14,5 @@ parameters:
     mailer_password: ~
 
     secret: EDITME
+    defuse_secret: def000003f125cbd07bfce7748bf501fa3f5ebb6b8a4d785ddb7b328948ea4e88973d5c1cdf79a5a2784b723ad2fb9d802111d73ec98be7ef55e47966a89e9b1fcfa35b0
```

Comment:
Let's say I used a default key for some time and then realized I had to change it. Now I have data encrypted with one key and I have to somehow decrypt it first and then crypt with a new secret. It could be tricky to do.